src/test/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlExtensionTest.java

```diff
@@ -101,11 +103,12 @@ public void testToStageArchiveConfiguration_emptyExtraFiles() {
     extension.setDockerDirectory(dockerDirectory);
     extension.setExtraFilesDirectories(Collections.emptyList());
 
-    AppYamlProjectStageConfiguration generatedConfig = extension.toStageArchiveConfiguration();
+    AppYamlProjectStageConfiguration generatedConfig = extension.toAppYamlProjectStageConfiguration();
     Assert.assertEquals(appEngineDirectory.toPath(), generatedConfig.getAppEngineDirectory());
     Assert.assertEquals(stagingDirectory.toPath(), generatedConfig.getStagingDirectory());
     Assert.assertEquals(artifact.toPath(), generatedConfig.getArtifact());
     Assert.assertEquals(dockerDirectory.toPath(), generatedConfig.getDockerDirectory());
     Assert.assertEquals(0, generatedConfig.getExtraFilesDirectory().size());
   }
+
```

Comment:
Intentional newline?

Reply:
probs not, I think its from where I was trying to write an uptodate check test.